test/example/test.ts

```diff
@@ -1,97 +0,0 @@
-/*
```

Comment:
It would be nice to keep this (at least the modified version of it) as a canonical example on how the library should be used.

Reply:
Done